doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -279,3 +279,9 @@ Example for the ``nrf52840dk_nrf52840`` target board and the :ref:`matter_lock_s
    :class: highlight
 
    west build -b nrf52840dk/nrf52840 -- -Dlock_SNIPPET=diagnostic-logs
+
+.. important::
+
+  You can enable diagnostic log support only by setting the snippet because it contains the crucial devicetree overlays.
```

Comment:
```suggestion
  To enable diagnostic log support, you must use the :ref:`diagnostic logs snippet <ug_matter_diagnostic_logs_snippet>`, which contains required devicetree overlays.
```
This this line should replace Line 235.

With this change, the `important` can be changed to `note` and moved before Line 274 (between the list of Kconfig options and the use instructions).